peachjam/models/document_access_group.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.auth.models import Group
+from django.db import models
+from django.urls import reverse
+
+
+class DocumentAccessGroup(models.Model):
+    group = models.OneToOneField(
+        Group,
+        on_delete=models.CASCADE,
+        verbose_name="group",
+    )
+    description = models.TextField(null=True, blank=True)
+
+    class Meta:
+        verbose_name = "document access group"
```

Comment:
translateable

Reply:
(I think we translate these for other models?)

Reply:
we do. done